perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -81,6 +81,19 @@ MultiObjectTracker::MultiObjectTracker(const rclcpp::NodeOptions & node_options)
   double publish_rate = declare_parameter<double>("publish_rate", 30.0);
   world_frame_id_ = declare_parameter<std::string>("world_frame_id", "world");
   bool enable_delay_compensation{declare_parameter("enable_delay_compensation", false)};
+  debug_flag_ = declare_parameter<bool>("debug_flag", false);
```

Comment:
You are right. My plan is to write out default parameters to [default_tracker_param.yaml](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/multi_object_tracker/config/default_tracker.param.yaml). It will involve many changes so let us discuss in another PR.